src/dataformat/classes.py

```diff
@@ -92,9 +94,19 @@ def create_all_teams(self):
             team_list.append(team)
         return team_list
 
+    def create_squad(self, list_of_player_names, list_of_player_teams):
+        if len(list_of_player_names) != 15:
+            print('Not enough players..')
+            exit(1)
```

Comment:
For further debugging, you may consider using another exit-number, and make a list storing the different exit-numbers used?